deployment/terraform/visualizer/templates/visualizer_conf.tpl

```diff
@@ -0,0 +1,12 @@
+visualizer {
+    metric-consumer {
+      bootstrap.servers = "${kafka_endpoint}"
+      group.id = "my-group"
+      topic = "anomalies"
+      key.deserializer = "org.apache.kafka.common.serialization.StringDeserializer"
+      value.deserializer = "com.expedia.adaptivealerting.kafka.serde.MappedMetricDataJsonSerde$Deser"
```

Comment:
These should all be configurable

Reply:
i was checking metric function app and seems like not all configs need to come from deployer since these configs dont change based on environment is prod or test. 
https://github.com/ExpediaDotCom/adaptive-alerting/blob/master/deployment/terraform/aa-metric-functions/templates/aa-metric-functions_conf.tpl#L5

Reply:
In principle those ought to be configurable per env actually. For Expedia we probably won't change the serializers anytime soon, but since you will be updating the group ID and the topic name (the topic name for sure needs to be configurable), it makes sense to just make the serializers configurable too. I know metric-functions isn't doing it, but I would ask that we make visualizer do it the right way so we can use visualizer as the gold standard instead of just copying the hardcodes in metric-functions.